spec/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -116,6 +117,7 @@ interface Packet {
 - The `sequence` number corresponds to the order of sends and receives, where a packet with an earlier sequence number must be sent and received before a packet with a later sequence number.
 - The `timeoutHeight` indicates a consensus height on the destination chain after which the packet will no longer be processed, and will instead count as having timed-out.
 - The `timeoutTimestamp` indicates a timestamp on the destination chain after which the packet will no longer be processed, and will instead count as having timed-out.
+- The `delayPeriod` indicates a period that must elapse after validation of a header before a packet can be processed. It is only safe to allow the user to set this field when using unordered channels.
```

Comment:
And what about ordered channels? How we ensure the same level of security for ordered channels?

Reply:
Ordered channels would need to apply some bound to the `delayPeriod` which their packets are allowed to set, or even fix a particular value (possibly `0` or a low value).